From acadda42f42f3f7fc460be623455b6984f6c624c Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:04:56 -0400
Subject: [powerpc] Update the 00-Index in Documentation/powerpc

Message-id: <1410545655-205645-68-git-send-email-dzickus@redhat.com>
Patchwork-id: 94102
O-Subject: [RHEL7 PATCH 067/626] powerpc: Update the 00-Index in Documentation/powerpc
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 380a0b3535b09ea6fe580f9cb6f28ca168f14abc
Author: Michael Ellerman <michael@ellerman.id.au>
Date:   Thu Aug 15 15:22:14 2013 +1000

    powerpc: Update the 00-Index in Documentation/powerpc

    People have been dropping things in here without updating the index, do
    it for them.

    Signed-off-by: Michael Ellerman <michael@ellerman.id.au>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Documentation/powerpc/00-INDEX b/Documentation/powerpc/00-INDEX
index 05026ce..6db73df 100644
--- a/Documentation/powerpc/00-INDEX
+++ b/Documentation/powerpc/00-INDEX
@@ -5,13 +5,20 @@ please mail me.
 
 00-INDEX
  - this file
+bootwrapper.txt
+ - Information on how the powerpc kernel is wrapped for boot on various
+   different platforms.
 cpu_features.txt
  - info on how we support a variety of CPUs with minimal compile-time
  options.
 eeh-pci-error-recovery.txt
  - info on PCI Bus EEH Error Recovery
+firmware-assisted-dump.txt
+ - Documentation on the firmware assisted dump mechanism "fadump".
 hvcs.txt
  - IBM "Hypervisor Virtual Console Server" Installation Guide
+kvm_440.txt
+ - Various notes on the implementation of KVM for PowerPC 440.
 mpc52xx.txt
  - Linux 2.6.x on MPC52xx family
 pmu-ebb.txt
@@ -19,3 +26,7 @@ pmu-ebb.txt
 qe_firmware.txt
  - describes the layout of firmware binaries for the Freescale QUICC
    Engine and the code that parses and uploads the microcode therein.
+ptrace.txt
+ - Information on the ptrace interfaces for hardware debug registers.
+transactional_memory.txt
+ - Overview of the Power8 transactional memory support.
-- 
1.7.1